STABLE_RELEASES.md

```diff
@@ -58,43 +58,53 @@ To smoothen the update to the latest stable release, the SDK includes a set of C
     * Bugs that may cause **loss of user's data**.
 * Other safe cases:
     * Bugs which don't fit in the aforementioned categories for which an obvious safe patch is known.
+    * Relatively small yet strictly non-breaking features with strong support from the community.
     * Relatively small yet strictly non-breaking changes that introduce forward-compatible client
     features to smoothen the migration to successive releases.
+    * Relatively small yet strictly non-breaking CLI improvements.
 
 ### What does not qualify as SRU
 
 * State machine changes.
-* New features that introduces API breakages (e.g. public functions removal/renaming).
+* Breaking changes in Protobuf definitions, as specified in [ADR-044](./docs/architecture/adr-044-protobuf-updates-guidelines.md).
+* Changes that introduces API breakages (e.g. public functions and interfaces removal/renaming).
+* Client-breaking changes in gRPC and HTTP request and response types.
+* CLI-breaking changes.
```

Comment:
We didn't decide if we're okay to backport CLI-breaking changes. I've heard both opinions, for and against.

My personal opinion is to forbid backporting CLI-breaking changes.

Reply:
I think lines 72, and 70 should have an exception carved out explicitly in the case of bugs

Reply:
@ValarDragon The way it's framed right now, 70-72 are breaking changes that should not go in a "Stable Release Update".

In case of bugs, they could exceptionally be handled by a special SRE case:

> In some circumstances, PRs that don't meet the aforementioned criteria might be raised and asked to be granted a *Stable Release Exception*.